cockroachcloud/managing-access.md

```diff
@@ -0,0 +1,404 @@
+{{site.data.alerts.end}}
+
+If you are sure you want to delete the organization, proceed with the following steps:
+
+1. Navigate to the **Settings** page for the organization you want to delete.
+1. Click **Delete**.
+1. Enter the name of the organization.
+1. Click **Delete**.
+
+    You will be automatically logged out of your account.
+
+## Manage service accounts
+
+{{site.data.alerts.callout_info}}
+The access management model for service accounts changed with release v23.1. Service accounts created prior to this release may still have the following legacy roles:
+
+- The `ADMIN` role  allows the service account full authorization for the organization, where the service account can create, modify, and delete clusters.
+- The `CREATE` role allows the service account to create new clusters within the organization.
+- The `DELETE` role allows the service account to delete clusters within the organization.
+- The `EDIT` role allows the service account to modify clusters within the organization.
+- The `READ` role allows the service account to get details about clusters within the organization.
+{{site.data.alerts.end}}
+
+### Create a service account
+
+1. On the **Access** page, select the **Service Accounts** tab.
+1. Click **Create**.
+1. Enter a **Name** and **Description**.
+1. Create and export an **API key**.
+1. Confirm creation of the service account.
+
+{{site.data.alerts.callout_info}}
+Service accounts, like users, are given only the organization member role default upon creation. This role grants no access.
+{{site.data.alerts.end}}
+
+### Edit roles on a service account
+
+1. On the **Access** page, select the **Service Accounts** tab.
+1. In the row for the target service account, click, click the three-dots **Action** button and select **Edit Roles**.
+1. Note that a number of fine-grained roles can be assigned to a given service account. These are the same [roles that can be assigned to users](authorization.html#organization-user-roles). Each role is represented by a row. Each row has a **scope**, which is either **Organization** or the name of a particular cluster. If the role is Cluster Administrator or Cluster Developer, giving it the scope of organization means that it applies to all clusters in the organization.
+
+### API access
+
+Each service account can have one or more API keys. API keys are used to authenticate and authorize service accounts when using the API. All API keys created by the account are listed under **API Access**.
+
+We recommend creating service accounts with the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege), and giving each application that accesses the API its own API key. This allows fine-grained permission and API key access to the cluster.
```

Comment:
Suggested this change because different API keys belonging to the same service account have the same access level, since roles are granted to the service account and not the API key.